### ISSUE TITLE
Fontello build task failing

### ISSUE THREAD

**Issue by jbeezley** [girder/girder#1268]
The grunt task `fontello:ext_font` is currently failing with the following stack trace:
```
Running "fontello:ext_font" (fontello) task
[D] Task source: /Users/jbeezley/git/girder/node_modules/grunt-fontello/tasks/fontello.js
Verifying property fontello.ext_font exists in config...OK
File: [no files]
Options: host="http://fontello.com", config="clients/web/fontello.config.json", fonts="clients/web/static/built/fontello/font", styles="clients/web/static/built/fontello/css", exclude=[], zip=false, scss=false, force, updateConfig=false
Verify paths...OK
[D] clients/web/static/built/fontello/font verified!
[D] clients/web/static/built/fontello/css verified!
Fetching archive...Creating session...OK
[D] sid: 95d43a8575e3b6fccfdc27fe9305aab9
Fatal error: Cannot read property 'config' of null
TypeError: Cannot read property 'config' of null
    at setSession (/Users/jbeezley/git/girder/node_modules/grunt-fontello/tasks/lib/fontello.js:33:43)
    at fetchStream (/Users/jbeezley/git/girder/node_modules/grunt-fontello/tasks/lib/fontello.js:129:3)
    at /Users/jbeezley/git/girder/node_modules/grunt-fontello/tasks/lib/fontello.js:106:12
    at done (/Users/jbeezley/git/girder/node_modules/needle/lib/needle.js:357:7)
    at PassThrough.<anonymous> (/Users/jbeezley/git/girder/node_modules/needle/lib/needle.js:546:9)
    at emitNone (events.js:73:20)
    at PassThrough.emit (events.js:167:7)
    at endReadableNT (_stream_readable.js:906:12)
    at nextTickCallbackWith2Args (node.js:455:9)
    at process._tickCallback (node.js:369:17)
```

After this happens, the fontello config file is mutated:
```
$ git diff
diff --git a/clients/web/fontello.config.json b/clients/web/fontello.config.json
index 4a52502..a374f6c 100644
--- a/clients/web/fontello.config.json
+++ b/clients/web/fontello.config.json
@@ -1,5 +1,5 @@
 {
-   "name": "3643f1665274345f0d7b88fbdb539555",
+   "name": "",
    "css_prefix_text": "icon-",
    "css_use_suffix":
```
after which the task will succeed.

**Comment by manthey** [comment]
A similar (the same) problem is discussed in [grunt-fontello issue #5](https://github.com/jubalm/grunt-fontello/issues/5).

**Comment by jbeezley** [comment]
Yeah, it's odd.  I'm not sure how the code there is supposed to work.  Callback methods seem to have the signature `foo(options, ...)`, but are called as `callback(null, options)`.

**Comment by manthey** [comment]
Resolved by PR #1269.
